sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -35,7 +50,12 @@ class JDBCOptions(
   val table = parameters("dbtable")
 
   // ------------------------------------------------------------
-  // Optional parameter list
+  // Optional parameters
+  // ------------------------------------------------------------
+  val driver = parameters.get(JDBCOptions.JDBC_DRIVER_CLASS)
```

Comment:
The naming is wrong. This is user-specified driver class. 

I think we can move the whole logics here to determine the actual JDBC driver. See `object JdbcUtils`